patternlab/styleguide/source/_patterns/03-organisms/by-author/account-manager.twig

```diff
@@ -1,6 +1,6 @@
 <section class="ma__account-manager">
   {# Static title, users can not change #}
-  <h3 id="manage-account" class="ma__comp-heading">Manage your account</h3>
+  <h3 id="manage-your-account" class="ma__comp-heading">Manage your account</h3>
```

Comment:
To make the heading level flexible depending on where it's placed in a page in case the template is used for other purposes, can you add the comp heading with include? So, the heading can have a proper heading level number based on where it's placed.

sample:  patternlab/styleguide/source/_patterns/03-organisms/by-author/contact-list.twig

```
  {% set contactHeading = 2 %}

  {% if contactList.compHeading %}
    {% set compHeading = contactList.compHeading %}
    {% include "@atoms/04-headings/comp-heading.twig" %}
    {% set contactHeading = (compHeading.level ? : contactHeading) + 1 %}
  {% endif %}
```
You can pass `id` value to `@atoms/04-headings/comp-heading.twig` according to `@atoms/04-headings/comp-heading.json`:
```
 {
  "compHeading": {
    "title": "Title text",
    "titleContext": "",
    "level": "",
    "color": "",
    "id": "GUID9827924",
    "centered": ""
  }
}
```